src/main.ts

```diff
@@ -474,7 +477,16 @@ export class Purchases {
    */
   @requiresLoadedResources
   public purchase(params: PurchaseParams): Promise<PurchaseResult> {
-    const { rcPackage, purchaseOption, htmlTarget, customerEmail } = params;
+    const {
+      rcPackage,
+      purchaseOption,
+      htmlTarget,
+      customerEmail,
+      selectedLocale = "en",
```

Comment:
are we using simple locales or `en_US` format?

Reply:
The Translator supports both, if it doesn't find `en_US` in the available locales it tries with `en`.